R/composite_ppi.R

```diff
@@ -74,11 +74,8 @@
 #' # across the available scans at each geographic location
 #' composite <- composite_ppi(ppis, method = "max", res=1000)
 #'
-#' # Download basemap
-#' bm <- download_basemap(composite)
-#'
 #' # Plot the calculated max product on the basemap
-#' map(composite, bm)
+#' map(composite)
```

Comment:
map's for composites should so locations for each radar in the composite (I noticed the unit test doesn't test for this functionality)
```
pvol1=read_pvolfile("~/Desktop/KBGM20230726_112800_V06")
pvol2=read_pvolfile("~/Desktop/KHGX20170504_013934_V06")
ppi1=project_as_ppi(pvol1$scans[[1]])
ppi2=project_as_ppi(pvol2$scans[[1]])
ppi3=composite_ppi(list(pp1,pp2))
map(ppi3) # should show two radar locations
```